recipes/ucsc-hggcpercent/meta.yaml

```diff
@@ -1,39 +1,43 @@
 {% set package = "ucsc-hggcpercent" %}
 {% set program = "hgGcPercent" %}
-{% set version = "377" %}
-{% set sha256 = "932f149c19641064a9cd3f2382cbb54b45a9292b8444792872d531346925d676" %}
+{% set version = "448" %}
+{% set sha256 = "90c28f06f3f5b6d79ff141cbf745e6a0c1b289a2ce640bbd1b0a14c826f08a85" %}
 
 package:
-  name: "{{ package }}"
-  version: "{{ version }}"
+  name: {{ package }}
+  version: {{ version }}
 
 source:
-  url: "http://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v{{ version }}.src.tgz"
-  sha256: "{{ sha256 }}"
+  url: http://hgdownload.cse.ucsc.edu/admin/exe/userApps.archive/userApps.v{{ version }}.src.tgz
+  sha256: {{ sha256 }}
   patches:
-    - "include.patch"
+    - include.patch
 
 build:
+  number: 0
   skip: True  # [osx]
-  number: 4
+  ignore_run_exports:
```

Comment:
This should probably be not added. Why do you think this is not needed?

Reply:
I have removed ignore_run_exports in my next commit. Thanks for catching this!